cmd/format.go

```diff
@@ -234,6 +236,43 @@ func createStorage(format meta.Format) (object.ObjectStorage, error) {
 			u.RawQuery = values.Encode()
 			format.Bucket = u.String()
 		}
+
+		// Configure client TLS when params are provided
+		caCerts := values.Get("ca-certs")
+		if os.Getenv("JFS_CA_CERTS") != "" {
```

Comment:
Since the ca could be used for meta-url or bucket, then `JFS_CA_CERTS` will be confusing, so it's better to don't support that.

Reply:
Would you like me to rename variables to JFS_BUCKET_ or you prefer to remove env vars at all? :)

I added them as an option, but i'm ok with query params only too

Reply:
It's better to remove them

Reply:
Done! :)